presto-main/src/test/java/com/facebook/presto/type/BenchmarkDecimalOperators.java

```diff
@@ -19,6 +19,7 @@
 import com.facebook.presto.operator.PageProcessor;
 import com.facebook.presto.spi.Page;
 import com.facebook.presto.spi.PageBuilder;
+import com.facebook.presto.spi.type.BigintType;
```

Comment:
To make reading the benchmark results easier, I'd suggest formatting them like this:

```
* Multiply Operator

Expression  Benchmark  Score
d1 * d2     Fast       ...    
d1 * d2     Bigint     ...
...
```

Or even:

```
* Multiply Operator

Expression  Bigint  Fast
d1 * d2     ...     ...
...
```

Basically, you want to make it easy to see the difference before and after the optimizations were made.

BTW, why is the set of benchmarks for Fast vs BigInteger different? I'd expect the same set of benchmarks to be run against both implementations.